src/main/java/edu/nicholaidudakiwwarrick/advancedjava/Employee.java

```diff
@@ -0,0 +1,48 @@
+package edu.nicholaidudakiwwarrick.advancedjava;
+
+/**
+ * Simple class to model an employee
+ *
+ * @author Nicholai Dudakiw-Warrick
+ */
+public class Employee {
+
+    private int employeeId;
+    private String employeeFirstName;
+    private String employeeLastName;
+
+    /**
+     * Creates a new employee instance
+     *
+     * @param employeeId            employee identification number
+     * @param employeeFirstName     employee first name
+     * @param employeeLastName      employee last name
+     */
+    public Employee (int employeeId, String employeeFirstName, String employeeLastName) {
```

Comment:
A slightly better example of dependency injection would be to use objects rather than primitives arguments. 
The objects should be declared as interfaces not concrete  classes.  This allows the caller to change the implementation type without having to also change this class. This is a very powerful construct.